### PR TITLE
[microk8s documentation] Documentation on clustering available with the 1.16 release

### PR DESCRIPTION
**Pull request by ktsakalozos**
This PR adds documentation on the clustering feature. This feature should go live with the 1.16 release planned for Sept 17.

@tvansteenburgh, lets see if this aligns with what we have planned.

### CODE REVIEW

File: docs/clustering.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: docs
+title: "Muli-node Clusters"
+---
+# Muli-node Clusters
```
Comment (evilnick) [inline docs/clustering.md]: Multi

**Comment by evilnick** [comment]
@ktsakalozos  okay, thanks i will have a run through this now

File: docs/clustering.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Join node with: microk8s.join ip-172-31-20-243:25000/DDOkUupkmaBezNnMheTBqFYHLWINGDbf
+
+If the node you are adding is not reachable through the default
+interface you can use one of the following:
+ microk8s.join 10.1.84.0:25000/DDOkUupkmaBezNnMheTBqFYHLWINGDbf
+ microk8s.join 10.22.254.77:25000/DDOkUupkmaBezNnMheTBqFYHLWINGDbf
+```
+
+The `add-node` command prints a `microk8s.join` command which should
+be executed on the MicroK8s instance that you wish to join to this
```
Comment (tvansteenburgh) [inline docs/clustering.md]: s/this/the/

File: docs/clustering.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+## Removing a node
+
+To remove a node from the cluster, use `microk8s.remove-node`:
+```
+> microk8s.remove-node 10.22.254.79
+```
+
+Finally, qn the removed node, run `microk8s.leave`. MicroK8s will restart
```
Comment (tvansteenburgh) [inline docs/clustering.md]: s/qn/on/

**Comment by ktsakalozos** [comment]
Any update on this?

**Comment by pmahnke** [comment]
@ktsakalozos it is unclear that this is ready for the webteam to push live... you need to change the labels AND comment that you are ok with it all.

**Comment by evilnick** [comment]
@pmahnke Due to recent permission changes across the webteam repos none of us can actually add or change any of the labels :crying_cat_face:

**Comment by ktsakalozos** [comment]
Apologies for the vague comment. Yes this is ready to go live as we recently released this feature.

Regarding the labels, I am not able to change them. That seems right since I am not familiar with the semantics of each label.

**Comment by pmahnke** [comment]
I will look into permissions

**Pull request merged**